### PR TITLE
Added auto package discovery for Laravel 5.5

### PR DESCRIPTION
**Pull request by keenminded**
Added auto package discovery for Laravel 5.5.

### CODE REVIEW

**Comment by semyonchetvertnyh** [comment]
@barryvdh could you approve this PR?

File: composer.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
     "require": {
         "php": ">=5.3.0",
         "illuminate/support": "5.0.* || 5.1.* || 5.2.* || 5.3.* || 5.4.* || 5.5.*",
-        "omnipay/common": "~2.0"
```
Comment (barryvdh) [inline composer.json]: This doesn't belong in this pr

Comment (keenminded) [reply composer.json]: @barryvdh Thank you for pointing this out. Reverted back to the original.
